src/base/system_parameter.c

```diff
@@ -8677,68 +8919,16 @@ prm_print (const SYSPRM_PARAM * prm, char *buf, size_t len, PRM_PRINT_MODE print
     {
       const KEYVAL *keyvalp = NULL;
 
-      if (intl_mbs_casecmp (prm->name, PRM_NAME_ER_LOG_LEVEL) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, er_log_level_words, DIM (er_log_level_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_LOG_ISOLATION_LEVEL) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, isolation_level_words, DIM (isolation_level_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_PB_DEBUG_PAGE_VALIDATION_LEVEL) == 0)
-	{
-	  keyvalp =
-	    prm_keyword (PRM_GET_INT (prm_value), NULL, pgbuf_debug_page_validation_level_words,
-			 DIM (pgbuf_debug_page_validation_level_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_HA_MODE) == 0
-	       || intl_mbs_casecmp (prm->name, PRM_NAME_HA_MODE_FOR_SA_UTILS_ONLY) == 0)
-
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, ha_mode_words, DIM (ha_mode_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_HA_SERVER_STATE) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, ha_server_state_words, DIM (ha_server_state_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_HA_LOG_APPLIER_STATE) == 0)
-	{
-	  keyvalp =
-	    prm_keyword (PRM_GET_INT (prm_value), NULL, ha_log_applier_state_words, DIM (ha_log_applier_state_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_COMPAT_MODE) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, compat_words, DIM (compat_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_CHECK_PEER_ALIVE) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, check_peer_alive_words, DIM (check_peer_alive_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_QUERY_TRACE_FORMAT) == 0)
-	{
-	  keyvalp =
-	    prm_keyword (PRM_GET_INT (prm_value), NULL, query_trace_format_words, DIM (query_trace_format_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_FAULT_INJECTION_TEST) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm_value), NULL, fi_test_words, DIM (fi_test_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_HA_REPL_FILTER_TYPE) == 0)
-	{
-	  keyvalp =
-	    prm_keyword (PRM_GET_INT (prm->value), NULL, ha_repl_filter_type_words, DIM (ha_repl_filter_type_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_TDE_DEFAULT_ALGORITHM) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm->value), NULL, tde_algorithm_words, DIM (tde_algorithm_words));
-	}
-      else if (intl_mbs_casecmp (prm->name, PRM_NAME_REGEXP_ENGINE) == 0)
-	{
-	  keyvalp = prm_keyword (PRM_GET_INT (prm->value), NULL, regexp_engine_words, DIM (regexp_engine_words));
-	}
-      else
+      switch (prm->id)
```

Comment:
```suggestion
      switch (SYSPRM_GET_ID(prm))
```
prm->id 로 접근하고 있는 부분에 대해서도 확인 부탁드립니다.

Reply:
SYSPRM_GET_ID() 매크로를 제거하고 prm->id로 표현하도록 하겠습니다.